.pre-commit-config.yaml

```
@@ -9,34 +9,46 @@ ci:
 
 repos:
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.0
+    rev: v2.31.0
```

Ok this highlights the need to upgrade the version in the repo where you've taken this code from as it's old.

```suggestion
    rev: v3.15.0
```